skopt/__init__.py

```diff
@@ -9,16 +9,6 @@
 
 ## Install
 
-These instructions will setup the latest released version of `scikit-optimize`.
-Currently `scikit-optimize` relies on a yet unreleased version of `scikit-learn`.
-This means you will have to install that version by hand and probably want to
-create a separate virtualenv or conda environment for it.
-
-```
-pip install -e git+https://github.com/scikit-learn/scikit-learn.git#egg=scikit-learn-0.18dev
-```
-
-After this you can install `scikit-optimize` with:
```

Comment:
```
conda create -n skopt-clean python=3.5
activate-it
pip install scikit-optimize
```

seems to work and correctly pulls in `scikit-learn-0.18`. So I think we can get rid of these instructions.